tools/confix/migrations.go

```diff
@@ -22,24 +23,98 @@ const (
 type MigrationMap map[string]func(from *tomledit.Document, to, planType string) transform.Plan
 
 var Migrations = MigrationMap{
-	"v0.45": NoPlan, // Confix supports only the current supported SDK version. So we do not support v0.44 -> v0.45.
-	"v0.46": PlanBuilder,
-	"v0.47": PlanBuilder,
-	"v0.50": PlanBuilder,
-	"v0.52": PlanBuilder,
+	"v0.45":    NoPlan, // Confix supports only the current supported SDK version. So we do not support v0.44 -> v0.45.
+	"v0.46":    PlanBuilder,
+	"v0.47":    PlanBuilder,
+	"v0.50":    PlanBuilder,
+	"v0.52":    PlanBuilder,
+	"serverv2": PlanBuilder,
```

Comment:
nit: I was thinking we could have a different plan builder for v2 actually.
This avoids making the one that works well for baseapp SDK more complicated.

Reply:
Understood. I made these changes to ensure compatibility with all versions without any issues. If a separate plan builder is required, please let me know and I can add it.